examples/spec/integration/register_namespace_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'workflows/timeout_workflow'
+
+describe 'Temporal.register_namespace' do
+  it 'can register a new namespace' do
+    # have to generate a new namespace on each run because currently can't delete namespaces
+    name = "test_namespace_#{SecureRandom.uuid}"
+    description = 'this is the description'
+    retention_period_days = 30
+    data = { test: 'value' }
+
+    Temporal.register_namespace(name, description, retention_period_days: retention_period_days, data: data)
```

Comment:
We probably need to cleanup this as it will pollute people's local Temporal setup with every test run

Reply:
You can't [delete namespaces ](https://community.temporal.io/t/how-to-delete-a-namespace/677)(at least not currently supported). Any ideas how we can clean it up?

Reply:
Oh wow, that's unfortunate. Let's make sure to follow up once Temporal has implemented it